application/azure/Dockerfile

```diff
@@ -0,0 +1,16 @@
+from mcr.microsoft.com/dotnet/core/sdk:3.1
+
+RUN mkdir azure
+WORKDIR /azure
+RUN dotnet new web
+COPY *.csproj *.cs *.nupkg ./
+RUN dotnet add package SecretHub -s $PWD
+
+# Install SecretHub
+RUN echo "deb [trusted=yes] https://apt.secrethub.io stable main" > /etc/apt/sources.list.d/secrethub.sources.list && apt-get update
+RUN apt-get install -y secrethub-cli
```

Comment:
Considering Docker's caching and layers system, it's better to first install any dependencies and then `COPY` source code:

```suggestion
# Install SecretHub
RUN echo "deb [trusted=yes] https://apt.secrethub.io stable main" > /etc/apt/sources.list.d/secrethub.sources.list && apt-get update
RUN apt-get install -y secrethub-cli

RUN mkdir azure
WORKDIR /azure
RUN dotnet new web
COPY *.csproj *.cs *.nupkg ./
RUN dotnet add package SecretHub -s $PWD
```

Reply:
Fixed